store/storage/sqlite/iterator.go

```diff
@@ -7,7 +7,6 @@ import (
 	"strings"
 
 	"golang.org/x/exp/slices"
-	_ "modernc.org/sqlite"
```

Comment:
Was this intended? Where does the `sql` package come from now?

Reply:
from `database/sql`, It seems like the the db instance is bind in `db.go`, anyhow the test works

Reply:
Interesting...ok, thanks!